grpc-1.70.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: grpc-1.70
   version: "1.70.1"
-  epoch: 2
+  epoch: 3
```

Comment:
We need to bump this epoch to build grpc using the latest protobuf version otherwise there are problems resolving the libprotobuf deps.